src/utils/productColors.js

```diff
@@ -53,7 +53,7 @@ const tableStyles = {
 
 const askWidths = {
   lojing: 'max(24rem, 30vw)',
-  alyson: 'min(100%, 24rem)',
+  alyson: '25rem',
```

Comment:
let's keep the min width check and not set it to static value 25rem because it will create a horizontal scroll bar in the small screens.

Reply:
The add another button is above the additional addresses on the figma, hence why I put it there. 
As for the width change, on several questions the title is so long that it wraps onto multiple lines and looks quite rough, and that min(100%) was causing that. Leaving it at a fixed value prevented this, though is not ideal